tests/dummy/app/controllers/share-discover-page.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+
+let filterQueryParams = ['types', 'provider', 'subject', 'tags', 'sources', 'publishers', 'funders', 'institutions', 'organizations', 'language', 'contributors', 'type'];
+
+export default Ember.Controller.extend({
+    contributors: '',
+    detailRoute: 'nodes.detail',
+    discoverHeader: 'Demonstration of discover-page component',
+    end: '',
+    i18n: Ember.inject.service(),
+    institutions: '',
+    filterMap: {
+        providers: 'sources',
+        subjects: 'subjects'
+    },
+    filterReplace: {
+        'Open Science Framework': 'OSF',
+        'Cognitive Sciences ePrint Archive': 'Cogprints',
```

Comment:
I do not believe we should translate these, but I will put up follow-up ticket for this and see what others think.